### PR TITLE
Fix CSP error for GTagMgr in base

### PR DESCRIPTION
**Pull request by jdfox**
# Motivation and Context
CSP script error in Tag Manager code, found during investigations with DST, complains about missing nonce (or alternatives) 

# What has changed
Added missing nonce to script tag in base html template

# How to test?
Deploy to dev, run acceptance tests, check for CSP error

# Links
- https://trello.com/c/etrcdyrU

# Screenshots (if appropriate):

### CODE REVIEW

File: frontstage/views/sign_in/sign_in.py [hunk 1]
```diff
@@ -91,8 +91,8 @@ def login():  # noqa: C901
         response.set_cookie('authorization',
                             value=session.session_key,
                             expires=session.get_expires_in(),
-                            secure=secure,
-                            httponly=secure)
+                            secure=True,
```
Comment (warrenbailey) [inline frontstage/views/sign_in/sign_in.py:94]: I think this is configurable for a reason??

The secure variable is set by the WTF_CSRF_ENABLED environment variable and it won't make a difference in preprod/prod as it's enabled, so I'm not sure why you're doing this?

**Review by warrenbailey — CHANGES_REQUESTED** [review]

File: frontstage/views/sign_in/sign_in.py (hunk 1 above)
Comment (jdfox) [reply frontstage/views/sign_in/sign_in.py]: You're right: reverting.

**Review by warrenbailey — APPROVED** [review]

**Pull request merged**